src/jarabe/journal/misc.py

```diff
@@ -429,4 +467,4 @@ def get_mount_color(mount):
     color = XoColor('%s,%s' %
                     (colors[index][0],
                      colors[index][1]))
-    return color
+    return color
```

Comment:
Again you removed the newline from end of file?  Why did you do that, or if a tool did it why did you let the change through into the commit?  Aren't you checking every change?